packages/openapi-ts/README.md

```diff
@@ -6,12 +6,12 @@
 
 ## Features
 
-- Generate TypeScript clients from OpenAPI v2.0, v3.0, and v3.1 specifications
-- Support JSON or YAML input files
-- Handle external references using [JSON Schema $Ref Parser](https://github.com/APIDevTools/json-schema-ref-parser/)
-- Generate Fetch, Node-Fetch, Axios, Angular, or XHR HTTP clients
-- Can be used with CLI, Node.js, or npx
-- Abortable requests through cancellable promise pattern
+-   Generate TypeScript clients from OpenAPI v2.0, v3.0, and v3.1 specifications
```

Comment:
Is Prettier adding a double space here? Seems odd

Reply:
I think that is like that for a reason. Ie its preferred formatting